mica/web/views.py

```diff
@@ -54,3 +56,48 @@ def get_context_data(self, **kwargs):
             context['pcad_data'] = pcad_data
 
         return context
+
+
+class TelemTableView(BaseView, TemplateView):
+    template_name = 'mica/telem_table.html'
+
+    def get_context_data(self, **kwargs):
+        # Call the base implementation first to get a context
+        context = super(TelemTableView, self).get_context_data(**kwargs)
+        context['errors'] = []
+
+        start_time = self.request.GET.get('start_time', None)
+        stop_time = self.request.GET.get('stop_time', None)
+        if start_time is not None:
+            try:
+                start_time = DateTime(start_time)
+                start_time.date
+            except:
+                start_time = None
+                context['start_time_date'] = ''
+        if stop_time is not None:
+            try:
+                stop_time = DateTime(stop_time)
+                stop_time.date
```

Comment:
I think I threw this in because I wasn't getting an error in the "try" block on an empty time (just got the error later when trying to use it).